.rubocop.yml

```diff
@@ -0,0 +1,54 @@
+AllCops:
+  Include:
+    - '**/*.rb'
+    - 'archive_posts.rake'
+  Exclude:
```

Comment:
As there's no 'archive_posts.rake' file within your project, this shouldn't be included here.